src/databricks/labs/ucx/cli.py

```diff
@@ -98,6 +98,19 @@ def skip(w: WorkspaceClient, schema: str | None = None, table: str | None = None
     return ctx.table_mapping.skip_schema(schema)
 
 
+@ucx.command
+def unskip(w: WorkspaceClient, schema: str | None = None, table: str | None = None):
+    """Create a unskip comment on a schema or a table"""
+    logger.info("Running unskip command")
+    if not schema:
```

Comment:
We are inconsistent with handling required cli parameters. Could you create a tech debt issue to resolve this?

Reply:
Issue [2740](https://github.com/databrickslabs/ucx/issues/2740) has been created